vital/types/covariance.h

```diff
@@ -211,6 +211,16 @@ typedef covariance_< 2, double > covariance_2d;
 typedef covariance_< 2, float > covariance_2f;
 typedef covariance_< 3, double > covariance_3d;
 typedef covariance_< 3, float > covariance_3f;
+
+typedef std::shared_ptr<covariance_2d> covariance_2d_sptr;
```

Comment:
I think we should remove these typedefs.  I don't think they are used now and we don't want to encourage shared_pointers to these types

Reply:
Agreed